jenkinsci/blueocean-plugin
# when a run is not replayable then do not show the replay button

# Description

When a run is not replayable then do not show the replay button

# Testing instructions
Start and stop a pipeline build very quickly and the run will be unplayable (this happens before Pipeline can store flow nodes). Increase your chances by making the number of executors zero.

The run should not have a rerun button.

See [JENKINS-42997](https://issues.jenkins-ci.org/browse/JENKINS-42997).

# Submitter checklist

# Reviewer checklist

PTAL @imeredith @cliffmeyers
